src/js/disability-benefits/526EZ/pages/veteranInfo.js

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
+
+import currentOrPastDateUI from '../../../common/schemaform/definitions/currentOrPastDate';
+import fullNameUI from '../../../common/schemaform/definitions/fullName';
+import ssnUI from '../../../common/schemaform/definitions/ssn';
+import { genderLabels } from '../../../common/utils/labels';
+
+import { veteranInformationViewField, getVerifiedChapterPair } from '../helpers';
+
+export default function createVeteranInfoChapter(formSchema, isReview) {
+
+  const { fullName, date } = formSchema.definitions;
+
+  const uiSchema = {
+    fullName: fullNameUI,
+    socialSecurityNumber: _.assign(ssnUI, {
+      'ui:title': 'Social Security number',
+      'ui:required': form => !form.vaFileNumber,
+    }),
+    vaFileNumber: {
+      'ui:title': 'VA file number',
+      'ui:required': form => !form.socialSecurityNumber,
```

Comment:
This is how we conditionally require ssn/vaFileNumber on other forms, depending on what is required and how, this may need to be updated.

Reply:
Should match that conditional behavior from other forms

Reply:
Thanks @goldenmeanie , in that case, shall we update the title to read:
ssn:
'Social Security number (must have this or a VA file number)'
vaFileNumber:
'VA file number (must have this or a Social Security number)'

Reply:
What do you think about adapting this pattern https://staging.vets.gov/education/apply-for-education-benefits/application/1990E/applicant/information?

**State 1**:
Social Security number {SSN field, **required**}
[ ] I don’t have a Social Security number

VA file number {text field}

**State 2**:
Social Security number {SSN field}
[**×**] I don’t have a Social Security number

VA file number {text field, **required**}